pkg/catalog/manager/crd.go

```diff
@@ -12,13 +13,23 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/apimachinery/pkg/util/validation"
 )
 
 const (
 	CatalogNameLabel  = "catalog.cattle.io/name"
 	TemplateNameLabel = "catalog.cattle.io/template_name"
 )
 
+type ForgettableError struct {
```

Comment:
Interesting idea here. I don't think this name is most appropriate though. Something more like `InvalidChartError` ?